src/main.cpp

```diff
@@ -4799,8 +4799,9 @@ bool SendMessages(CNode* pto, bool fSendTrickle)
             BOOST_FOREACH(const CAddress& addr, pto->vAddrToSend)
             {
                 // returns true if wasn't already contained in the set
```

Comment:
This comment is no longer relevant: `contains` is self-documenting in contrast to `insert`'s second return value